src/rooms/rooms.service.ts

```diff
@@ -64,7 +64,7 @@ export class RoomsService {
                          {
                                 if(imgLink.includes(s3Urls[k]))
                                 {
-                                    replaceLink = imgLink.replace(s3Urls[k],coronasafe_cdn);
+                                    replaceLink = imgLink.replace(`https://${s3Urls[k]}/`,"");
```

Comment:
The same logic is used in multiple places, It should be consolidated as a function for future usability